build/IceRpc.Version.props

```diff
@@ -2,5 +2,8 @@
     <PropertyGroup>
         <!-- The version of the IceRPC assembly files  -->
         <Version Condition="'$(Version)' == ''">0.3.1-preview1</Version>
+        <!-- The Protobuf version used by IceRpc.Protobuf -->
```

Comment:
I move Protobuf version here to ensure that `IceRpc.Protobuf`, `IceRpc.Protobuf.Tools`, and `IceRpc.ProtoGen` all use the same protobuf version.